src/assertions/there.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function there(client, chai, utils) {
```

Comment:
Yeah I personally don't like the the dom flag because it's not very semantic. I guess the only thing holding me back from merging this PR is that I think the ```there``` assertion name is not very semantic